.github/workflows/release-supermarket.yml

```diff
@@ -0,0 +1,21 @@
+name: chef supermaket
+on:
+  release:
+    types: [published]
+
+jobs:
+  share:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+      - uses: ruby/setup-ruby@v1  
+        with:
+          ruby-version: '3.0' # Not needed with a .ruby-version file
```

Comment:
might as well make this 3.1 to buy another year before you need to update things